.github/workflows/compile.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-20.04, macos-latest, windows-latest]
```

Comment:
You should still test Ubuntu latest apart from 20.04

Reply:
Makes sense, I can add it back to the matrix and skip the binary upload in that case ?